config.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"encoding/json"
+	"log"
+	"os"
+)
+
+// Config TODO
+type Config struct {
+	CustomerName string
+	Directory    map[string]interface{}
+	Slack        slackConfig
```

Comment:
public struct with private type is quite limited in its use, I would pick either public or private type and name.

Reply:
fixed.